test/embed.test.js

```diff
@@ -17,10 +17,17 @@
 const assert = require('assert');
 const path = require('path');
 const NodeHttpAdapter = require('@pollyjs/adapter-node-http');
+const proxyquire = require('proxyquire');
 const FSPersister = require('@pollyjs/persister-fs');
 const { setupMocha: setupPolly } = require('@pollyjs/core');
-const { embed } = require('../src/embed.js');
 const { assertContains } = require('./utils');
+const testFetch  = require('@adobe/helix-fetch').context({
+  httpsProtocols: ['http1'],
+  httpProtocols: ['http1'],
+}).fetch;
+const { disconnectAll } = require('@adobe/helix-fetch');
+
+const { embed } = proxyquire('../src/embed', { './unsplash': proxyquire('../src/unsplash.js', { '@adobe/helix-fetch': { fetch : (url) => testFetch(url) } })});
```

Comment:
Better to use `helix-fetch` in force-HTTP1 mode for tests.

Reply:
that's what I do; testFetch is in force http1 mode; but embed doesn't know that unless I'm missing something?